### DISCUSSION TITLE
Add outside collaborators to a team without giving them acess to other repos in an organization

### DISCUSSION THREAD

**Discussion by aidan-mundy** [community/community#6404] [Other Feature Feedback, Questions, & Ideas]
This is a feature that is highly important and should already exist. There is a thread [(of the same name)](https://github.community/t/add-outside-collaborators-to-a-team-without-giving-them-acess-to-other-repos-in-an-organization/2396) on the community forum that hasn't gotten much love from GitHub Staff, hoping that posting it here might put it in front of the right person.

**Comment by pattacini** [comment]
While GitHub is working on this, hopefully, you may find useful the workflow we have been using effectively for quite some time by now: https://github.com/icub-tech-iit/outside-collaborators.

The workflow allows for handling outside collaborators organization-wide.

**Comment by ghost** [comment]
@pattacini 
It would be really great news if GitHub is working on this, but I have seen no feedback from GitHub that they are. Perhaps someone from GitHub staff can confirm it? The thread at github community was started aug 2019, and still today the reply from GitHub is absent.

**Reply by ericb-summit** [reply]
For enterprise adoption this is a massive impediment. I would get laughed out of any security meeting to say I'm managing per-user permissions on a per-repository basis, manually. Like it's 1990. And like it's sustainable or responsible.

The only "workarounds" are third party integrations, that, given the security impact, I now need to own and maintain.

As a side comment, it's funny that Github provides, for countless projects, a clear concise means of reporting and tracking issues, but as a paying Github customer we're left posting on "community" or "discussions". You can be sure ignoring enterprise requirements will just drive people elsewhere, I know we are.

**Comment by AndrWeisR** [comment]
I am new to setting up Enterprise GitHub, and one of the first things I tried to do was to organise my outside collaborators into teams. It seems quite unbelievable that this can't be done. I agree 100% with what @ericb-summit said.

**Comment by atsju** [comment]
Same here. I want to be able to add outside collaborator to a team without giving accès to ALL repositories in the organization

**Comment by ssyberg** [comment]
+1

**Comment by YugoHino** [comment]
+1

**Comment by lubianat** [comment]
+1

**Comment by phage-zn** [comment]
++ just another instance of github not doing what the community needs/wants it seems

**Comment by RaresSalcudean** [comment]
+1

**Comment by gigadie** [comment]
This is absolutely needed!! +1
